Applications/Utils/MeshEdit/CreateBoundaryConditionsAlongPolylines.cpp

```diff
@@ -65,30 +65,61 @@ void convertMeshNodesToGeometry(std::vector<MeshLib::Node*> const& nodes,
 	geometry_sets.addPointVec(std::move(pnts), geo_name, pnt_names);
 }
 
-void writeBCsAndGeometry(GeoLib::GEOObjects & geometry_sets,
-	std::string & geo_name, bool write_gml)
+void writeGroundwaterFlowPointBC(std::ostream& bc_out,
+                                 std::string const& pnt_name, double head_value)
+{
+	bc_out << "#BOUNDARY_CONDITION\n";
+	bc_out << "  $PCS_TYPE\n";
+	bc_out << "    GROUNDWATER_FLOW\n";
+	bc_out << "  $PRIMARY_VARIABLE\n";
+	bc_out << "    HEAD\n";
+	bc_out << "  $GEO_TYPE\n";
+	bc_out << "    POINT " << pnt_name << "\n";
+	bc_out << "  $DIS_TYPE\n";
+	bc_out << "    CONSTANT " << head_value << "\n";
+}
```

Comment:
For next PR: The output should be collected in one place. Currently there are three places (files) with similar outputs.